### PR TITLE
Upp Dzho Automaton/Fighting Joe

### PR DESCRIPTION
**Pull request by Blundir**
# About the pull request

Adds the UPP Working Joe, aka Dzho Automaton (offical upp name), aka Fighting Joe (uscm nickname).

Fully based on what [Colonial Marines Operations Manual](https://avp.fandom.com/wiki/Colonial_Marines_Operations_Manual) has to say, as well as information from the  [AVP wiki page](https://avp.fandom.com/wiki/Fighting_Joe).

Main gimmick of these Joes, is that they are capable of using firearms and performing security/brig duties, also they are aggressive to english speakers when forced to speak english.

Added 128 voice lines in Russian just for this Joe subtype.

Added unique uniform made by Crowford, discord tag **sir.crow** .

The role is whitelisted under same conditions as normal Working Joe.

Roundstart spawners are supposed to be placed on https://github.com/cmss13-devs/cmss13/pull/7525, in the 1VAN/3 core.

image

# Explain why it's good for the game

New lore correct content is always good.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

https://github.com/user-attachments/assets/134fd53f-a8c1-4b62-b2f2-3b52a3be3df9

</details>

# Changelog
:cl:
add: Upp Dzho Automaton/Fighting Joe and his uniform
fix: fixed working joe not having eye intent set by default
/:cl:

### CODE REVIEW

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

File: code/modules/mob/living/carbon/human/species/working_joe/notice.dm [hunk 1]
```diff
@@ -75,13 +87,21 @@
 	emote_type = EMOTE_AUDIBLE|EMOTE_VISIBLE
 	joe_flag = WORKING_JOE_EMOTE|HAZARD_JOE_EMOTE
 
+/datum/emote/living/carbon/human/synthetic/working_joe/notice/species_upp
+	key = "speciesupp"
+	upp_joe_sound = 'sound/voice/joe/upp_joe/neizvestnoesuchestvo.ogg'
+	say_message = "Uknown creature."
```
Comment (Ben10083) [inline code/modules/mob/living/carbon/human/species/working_joe/notice.dm:93]:
```suggestion
	say_message = "Unknown creature."
```

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by harryob — APPROVED** [review]

**Pull request merged**